source/threads.tex

```diff
@@ -3666,7 +3666,7 @@
 
 \item these actions will not block for the shared state to become ready, except that it
 may block if all of the following are true: the shared state was created by a call to
-\tcode{std::async}, the shared state is not yet ready, and this was the last reference
```

Comment:
I think we've generally stayed away from anything `async`, because SG1 has been crafting that wording painstakingly and with great deliberation. For example, there's a case to be made that it should be easy for the reader to disinguish `std::async` from `std::launch::async`.